fishtank/src/cluster.test.ts

```diff
@@ -2,11 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import { ConfigOptions } from '@ironfish/sdk'
-import { promises } from 'fs'
+import { existsSync, promises } from 'fs'
```

Comment:
Let's avoid using synchronous functions in asynchronous code

Reply:
exist async is deprecated. The closest thing for async is access, but I would like to use exist/exsitSync since its straightforward